src/resolve/resolved-dns-rr.c

```diff
@@ -2097,23 +2097,27 @@ DnsSvcParam *dns_svc_params_copy(DnsSvcParam *first) {
         return copy;
 }
 
-int dns_txt_item_new_empty(DnsTxtItem **ret) {
+
+int dns_txt_item_new(DnsTxtItem **ret, size_t len) {
```

Comment:
Our current coding style suggests that the return parameter is always put last.

Alternatively, for functions where the only error code that might possibly be returned is -ENOMEM it might even be OK to just return the object directly as return value, instead of using a return a call-by-reference return parameter. But that's up to you.